src/service/extension-location.js

```diff
@@ -49,13 +49,19 @@ export function calculateExtensionScriptUrl(location, extensionId, isLocalDev) {
  * @param {string} entryPoint
  * @param {boolean=} isLocalDev
  * @param {boolean=} opt_rtv
+ * @param {boolean=} opt_minified
  * @return {string}
  */
 export function calculateEntryPointScriptUrl(
-    location, entryPoint, isLocalDev, opt_rtv) {
+    location, entryPoint, isLocalDev, opt_rtv, opt_minified) {
   const base = calculateScriptBaseUrl(location, isLocalDev);
-  if (opt_rtv) {
+  if (opt_rtv && opt_minified) {
     return `${base}/rtv/${getMode().rtvVersion}/${entryPoint}.js`;
+  } else if (opt_rtv) {
+    return `${base}/rtv/${getMode().rtvVersion}/${entryPoint}.max.js`;
```

Comment:
It fetches the current RTV file instead of the un-versioned file, e.g. `cdn.ampproject.org/rtv/123/v0.js` vs. `cdn.ampproject.org/v0.js`. We should never fetch max AND RTV though.